docs/references/ic-interface-spec.md

```diff
@@ -751,7 +758,7 @@ In order to read parts of the [The system state tree](#state-tree), the user mak
 
 -   `request_type` (`text`): Always `read_state`
 
--   `sender`, `nonce`, `ingress_expiry`: See [Authentication](#authentication)
+-   `sender`, `nonce`, `ingress_expiry`: See [Authentication](#authentication). `ingress_expiry` refers to this request's expiry, not the expiry of an earlier call request.
```

Comment:
This request might not refer to any earlier call request in which case the note can be confusing.